deep/settings.py

```diff
@@ -198,6 +198,9 @@
         'NAME': 'django.contrib.auth.password_validation.'
         'NumericPasswordValidator',
     },
+    {
+        'NAME': 'user.validators.CustomMaximumLengthValidator',
+    },
```

Comment:
`AUTH_PASSWORD_VALIDATORS` is only used on Django password create/change level https://docs.djangoproject.com/en/3.1/topics/auth/passwords/#module-django.contrib.auth.password_validation

We also need to check this when a user is created through API and when a user tries to login.